docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/BucketExact.tsx

```diff
@@ -0,0 +1,15 @@
+import { FileUploader } from '@aws-amplify/ui-react-storage';
+
+export const App = () => {
+  return (
+    <FileUploader
+      acceptedFileTypes={['image/*']}
+      bucket={{
+        bucketName: 'my-bucket-96e87892835c413e9963f3004a44e1ff',
```

Comment:
Note: this is not an actual bucket name

Reply:
Can you add a note so we don't wonder about that later, or make it more obvious like "my-bucket-xxxxxxx"

Reply:
Yep, I like going with "my-bucket-xxxxxxxx"